ports/esp8266/README.md

```diff
@@ -23,6 +23,9 @@ Supported features include:
 
 Documentation is available at http://docs.micropython.org/en/latest/esp8266/quickref.html.
 
+The default build requires a 2MiB flash chip, but see below for support for
+1MiB and 512kB options.
```

Comment:
There are quite a few inconsistent uses of kB vs KB vs KiB vs kiB in this commit.  They should all be kiB.

Reply:
Done